create_grids/incident/create_reduced_grid.py

```diff
@@ -0,0 +1,128 @@
+"""
+Code to make a reduced grid, for example limiting the number of age bins to a
+specific set or a max age.
+"""
+
+import numpy as np
+import argparse
+from pathlib import Path
+import yaml
+import h5py
+
+# synthesiser modules
+from synthesizer.abundances import Abundances
+from synthesizer.grid import Grid
+from synthesizer.photoionisation import cloudy17 as cloudy
+
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description='Reduce a grid')
+    
+   
+    # path to synthesizer_data_dir
```

Comment:
can you add these comments directly to each argument under the `help` keyword in `add_argument`? (see [here](https://docs.python.org/3/library/argparse.html#example))